atlas-commerce-connector/introduction.mdx

```diff
@@ -0,0 +1,16 @@
+---
+slug: /atlas-commerce-connector/introduction
+title: Atlas Commerce Connector for BigCommerce - Introduction
+description: Connect BigCommerce product data to WordPress.
```

Comment:
Are these supposed to be rendered? I understood those would work like metadata, but the GitHub preview displays them as a table.

Reply:
Yeah they will be rendered by developers.wpengine.com see Atlas Content Modeler for an example which is the dir I was going off: https://developers.wpengine.com/docs/atlas-content-modeler/introduction 

There are also contribution guides here: 
https://github.com/wpengine/headless-docs/blob/develop/CONTRIBUTING.md